src/main/scala/subsystem/Configs.scala

```diff
@@ -307,7 +308,7 @@ class WithFPUWithoutDivSqrt extends Config((site, here, up) => {
 })
 
 class WithBootROMFile(bootROMFile: String) extends Config((site, here, up) => {
-  case BootROMParams => up(BootROMParams, site).copy(contentFileName = bootROMFile)
+  case BootROMLocated(x) => up(BootROMLocated(x), site).map(_.copy(contentFileName = bootROMFile))
```

Comment:
should this automatically set BootROMLocated(...) to false?

Reply:
Not sure what you mean by this

Reply:
Ah, well, `WithBootROMFile` as written doesn't add a boot rom; if there is a boot rom in `up` it changes what file it uses to fill in the contents; but if there isn't one it does nothing. Your point is well taken though, since there are now some conjoined parameter settings I can make a `WithBootRom` that sets both

Reply:
ah, yeah i see. Ok, well you can ignore my comment here then